cpp/oneapi/dal/algo/decision_forest/test/batch.cpp

```diff
@@ -418,7 +426,13 @@ DF_BATCH_REG_TEST_NIGHTLY_EXT("df reg default flow") {
     const auto [data, data_test, checker_list] =
         this->get_reg_dataframe(wl.ds_info.name, wl.required_mse, wl.required_mae);
 
+    const splitter_mode splitter_mode_val =
```

Comment:
Do you have CI jobs that runs these test cases? The same question for internal CI. Do we validating it on GPU?

Reply:
Hi, I have shared the private CI run for you.